apps/examples/testcase/le_tc/network/itc_net_send.c

```diff
@@ -0,0 +1,185 @@
+/****************************************************************************
+*
+* Copyright 2017 Samsung Electronics All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+* either express or implied. See the License for the specific
+* language governing permissions and limitations under the License.
+*
+****************************************************************************/
+
+// @file itc_net_send.c
+// @brief ITC Test Case Example for send() API
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <pthread.h>
+
+#include "tc_internal.h"
+
+#define PORTNUM 1110
+#define MAXRCVLEN 20
+
+static pthread_mutex_t g_lock = PTHREAD_MUTEX_INITIALIZER;
+static pthread_cond_t g_cond;
```

Comment:
where is initialization?

Reply:
pthread_cond_t initialization raising the build issue 
"error: large integer implicitly truncated to unsigned type [-Werror=overflow]"

Reply:
Fixed review comments